frontend/src/pages/PipelineList.test.tsx

```diff
@@ -467,142 +485,13 @@ describe('PipelineList', () => {
       content:
         'Failed to delete pipeline: test-pipeline-id0 with error: "woops, failed!"\n\n' +
         'Failed to delete pipeline: test-pipeline-id1 with error: "woops, failed!"',
-      title: 'Failed to delete 2 pipelines',
+      title: 'Failed to delete some pipelines and/or some pipeline versions',
     });
 
     // Should show snackbar for the one successful deletion
     expect(updateSnackbarSpy).toHaveBeenLastCalledWith({
-      message: 'Delete succeeded for 2 pipelines',
+      message: 'Deletion succeeded for 2 pipelines',
```

Comment:
Can you add a case for deleting a pipeline and a version together? Because that logic is very complex. I want to make sure it is covered.